src/hotspot/share/interpreter/bytecodeTracer.cpp

```diff
@@ -96,7 +96,8 @@ class BytecodePrinter {
   // the adjustments that BytecodeStream performs applies.
   void trace(const methodHandle& method, address bcp, uintptr_t tos, uintptr_t tos2, outputStream* st) {
     ResourceMark rm;
-    if (_current_method != method()) {
+    bool methodChange = _current_method != method();
```

Comment:
nit, can you change this variable name to method_changed for the C++ coding style.

Reply:
Done!